src/app/metric_labels.rs

```diff
@@ -1,6 +1,5 @@
 use std::{
```

Comment:
Very minor style nit, so TIOLI, but this can now just be
```rust
use std::fmt::{self, Write};
```
(the multi-line import is no longer necessary since the `std::net` import was removed)